Tools/CISupport/ews-build/steps.py

```diff
@@ -3073,15 +3073,6 @@ def getResultSummary(self):
         return {'step': message}
 
 
-class RunWebKitPyPython2Tests(WebKitPyTest):
-    language = 'python2'
-    name = 'webkitpy-tests-{}'.format(language)
-    description = ['webkitpy-tests running ({})'.format(language)]
-    jsonFileName = 'webkitpy_test_{}_results.json'.format(language)
-    logfiles = {'json': jsonFileName}
-    command = ['python', 'Tools/Scripts/test-webkitpy', '--verbose', '--json-output={0}'.format(jsonFileName)]
-
-
 class RunWebKitPyPython3Tests(WebKitPyTest):
```

Comment:
We can consider removing this class as well, and just modify base class appropriately.